packages/expo-cli/src/commands/run/index.ts

```diff
@@ -0,0 +1,53 @@
+import { Command } from 'commander';
+
+import CommandError from '../../CommandError';
+import prompt from '../../prompts';
+import buildAndroidClientAsync from './buildAndroidClientAsync';
+
+type Options = {
+  platform?: string;
+  buildVariant?: string;
+};
+
+async function runAsync(projectRoot: string, options: Options) {
+  const platform = options.platform?.toLowerCase() ?? (await promptForPlatformAsync());
+  if (platform === 'android') {
+    if (typeof options.buildVariant !== 'string') {
+      throw new CommandError('--build-variant must be a string');
+    }
+    await buildAndroidClientAsync(projectRoot, {
+      buildVariant: options.buildVariant,
+    });
+  } else {
+    throw new Error('platform not implemented');
+  }
+}
+
+async function promptForPlatformAsync(): Promise<'android' | 'ios'> {
+  const { platform } = await prompt({
+    type: 'select',
+    message: 'Choose platform',
+    name: 'platform',
+    choices: [
+      {
+        title: 'Android',
+        value: 'android',
+      },
+      {
+        title: 'iOS',
+        value: 'ios',
+      },
+    ],
+  });
+  return platform;
+}
+
+export default function (program: Command) {
+  program
+    .command('run [path]')
+    .helpGroup('experimental')
```

Comment:
```suggestion
    .helpGroup('internal')
```
This will hide the command from `--help`